src/main/java/net/coderbot/iris/pipeline/ShaderPipeline.java

```diff
@@ -18,25 +18,29 @@
  */
 public class ShaderPipeline {
```

Comment:
The fields here should stay final, and you should just create a new ShaderPipeline instance instead externally.